rootfs/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.3
+
+RUN apk add --update-cache curl bash openssl sudo python3 && rm -rf /var/cache/apk/*
+RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN curl -sSL https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py | python -
+RUN pip install --disable-pip-version-check --no-cache-dir docker-py==1.6.0
+RUN pip install boto3
```

Comment:
This should be pinned to a specific version, such as `pip install boto3==1.2.3` (the current version on [pypi](https://pypi.python.org/pypi/boto3)). 

It is annoying to have to update such package versions from time to time, but it is more important to avoid surprises by locking down the tested versions of all dependencies.